sway-book/src/documentation/operations/assertions/index.md

```diff
@@ -0,0 +1,14 @@
+# Assertions
+
+An assertion is a condition which must evaluate to the [boolean](../../language/built-ins/boolean.md) value of `true` and its purpose is to prevent undesirable computation when the condition is evaluated to `false`.
```

Comment:
```suggestion
An assertion is a condition which must evaluate to the [Boolean](../../language/built-ins/boolean.md) value of `true` and its purpose is to prevent undesirable computation when the condition is evaluated to `false`.
```